src/prpy/planning/retimer.py

```diff
@@ -156,11 +157,9 @@ def __init__(self, do_blend=True, do_shortcut=True, blend_radius=0.5,
 
     @PlanningMethod
     def RetimeTrajectory(self, robot, path, options=None, **kw_args):
-        from copy import deepcopy
-        if options is None:
-            options = {}
-        new_options = deepcopy(options)
-        new_options['time_limit'] = kw_args.get('timelimit', 3.)
+        new_options = deepcopy(options) if options else dict()
```

Comment:
I don't understand what the point of the `options` argument is at all. Why aren't we simply using `kw_args` for this?

@psigen, I know that you didn't add this in the pull request, but do you understand what the logic is here?